File: trunk/quests/qeynos/Bassanio_Weekin.pl
sub EVENT_SIGNAL {
  if($signal == 1) {
	quest::start(58);
	}
  
  if($signal == 2) {
    quest::start(32);
	}
}


sub EVENT_SAY { 
if($text=~/Hail/i){
quest::say("Go to the taverns if you're looking for talk."); }
}
#END of FILE Zone:qeynos  ID:1103 -- Bassanio_Weekin 

File: trunk/quests/qeynos/Bassanio_Weekin.lua
function event_signal(e)
	if(e.signal == 1) then
		eq.stop();
	elseif(e.signal == 2) then
		eq.start(58);
	end
end

function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("Go to the taverns if you're looking for talk.");
	end
end
